packages/neos-ui-redux-store/src/CR/Nodes/selectors.js

```diff
@@ -198,6 +198,15 @@ export const makeGetAllowedChildNodeTypesSelector = (nodeTypesRegistry, elevator
 export const makeGetAllowedSiblingNodeTypesSelector = nodeTypesRegistry =>
     makeGetAllowedChildNodeTypesSelector(nodeTypesRegistry, parentNodeContextPath);
 
+export const makeHasAnyNodeTypeOptionsSelector = nodeTypesRegistry => createSelector(
```

Comment:
I think the name `HasAnyNodeTypeOptions` is somehow misleading. I'd suggest we use `isAllowedToAddChildOrSiblingNodes`

(also change at the other places)